integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/EnterAmountToTransferFragment.kt

```diff
@@ -228,4 +227,11 @@ class EnterAmountToTransferFragment : Fragment(R.layout.enter_amount_to_transfer
             }
         }
     }
+
+    fun showKeyboardAndButton(isVisible: Boolean) {
+        // avoid IllegalStateException
+        if (isAdded) {
+            viewModel.keyboardStateCallback.value = isVisible
+        }
+    }
```

Comment:
While the wallet is syncing, this does not work to hide the keyboard and button if the user leaves the Transfer Dash screen and returns to it.